checkpoints-relayer/src/sync_update.rs

```diff
@@ -0,0 +1,158 @@
+use super::*;
+use checkpoint_light_client_io::sync_update::Error;
+
+pub fn spawn_receiver(
+    client_http: Client,
+    beacon_endpoint: String,
+    sender: Sender<SyncCommitteeUpdate>,
+    delay: Duration,
```

Comment:
Why do you need to pass it as argument? You can just define it as a constant here or in `main`

Reply:
what is the problem with an argument?

Reply:
It's not clear what `delay` means and also calling side don't actually care about this `delay`